src/middleware/error-handler/index.js

```diff
@@ -0,0 +1,25 @@
+import { getConfig, getContainer, bind } from '@globality/nodule-config';
+
+export const DEFAULT_ERROR_MESSAGE = 'Something went wrong';
+
+// eslint-disable-next-line no-unused-vars
+export default function errorHandler(err, req, res, next) {
```

Comment:
**Nit:**

You could leave off the `next` argument rather than adding the `//eslint-disable` rule.

Reply:
If I do that it wouldn't be an error handler middleware as only 3 arguments would be defined. Functionality won't work

Reply:
Ahhhh ok, didn't realise our middleware required 4 args. Fair enough, ignore me then 😄